psi_cardinality/python/tests/BUILD

```diff
@@ -0,0 +1,25 @@
+load("@org_openmined_psi_python_deps//:requirements.bzl", "requirement")
+
+py_test(
+    name = "tests",
+    srcs = ["tests.py"],
+    main = "tests.py",
+    deps = [
+        "//psi_cardinality/python:psi_cardinality",
+        requirement("pytest"),
+    ],
+    python_version = "PY3",
+    srcs_version = "PY3",
+)
+py_test(
```

Comment:
Coult this be a py_binary? Then we could run tests with `bazel test //psi_cardinality/python/...` similar to other languages.

Also I'm not sure tests and benchmarks need an extra directory. At least for C++ and Go we kept unit tests next to the classes. But if it's a python convention to have these in an extra directory then I'm fine with it.